src/Motion.js

```diff
@@ -1,9 +1,9 @@
 export function managePerimeterCollision(state, obj){
   if(atLowerBound(state.screen, obj)){
-    state.currentScore += 10;
+    if(state.inGame){state.currentScore += 10}
     obj.gravity = false;
   } else if (atUpperBound(obj)) {
-    state.currentScore += 10;
+    if(state.inGame){state.currentScore += 10}
```

Comment:
i does add 10 to the score. I need to do a little research. I've found that when updating state this syntax works, but when actually setting a new attribute on state you must use `setState` @amaxwellblair